### PR TITLE
Update dlp-copy-matched-items-get-started.md

### PR DESCRIPTION
**Pull request by LucianFrango**
Updates to documentation based on customer feedback. CC @chrfox and @kmkoenen.

Additional detail added to page for clarity based on recent customer feedback as learning documentation around 3x points;
1) Licensing and Subscriptions
2) Setup Azure storage
3) Endpoint DLP settings configuration

### CODE REVIEW

File: microsoft-365/compliance/dlp-copy-matched-items-get-started.md [hunk 1]
```diff
@@ -39,7 +39,9 @@ Before you start these procedures, you should review [Learn about evidence colle
 
 ## Licensing and Subscriptions
 
-See the [licensing requirements for Information Protection](/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance#information-protection) for details on the subscriptions that support DLP. You don't need any additional licenses over what is needed for endpoint DLP.
+See the [licensing requirements for Information Protection](/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance#information-protection) for details on the subscriptions that support DLP.
+
+See the pre-requisite [licensing requirements for Azure Active Directory P1 or P2](azure/active-directory/roles/custom-create#prerequisites) needed to create custom Role-Based Access Control (RBAC) roles.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-copy-matched-items-get-started.md]:
```suggestion
See the pre-requisite [licensing requirements for Azure Active Directory P1 or P2](/azure/active-directory/roles/custom-create#prerequisites) needed to create custom role-based access control (RBAC) roles. 
```

File: microsoft-365/compliance/dlp-copy-matched-items-get-started.md [hunk 2]
```diff
@@ -75,14 +77,16 @@ You can also configure immutability policies for your blob data that protects ag
 The procedures for setting up your Azure storage account, container and blobs are documented in the Azure document set. Here are links to relevant articles you can refer to help you get started:
 1. [Introduction to Azure Blob Storage](/azure/storage/blobs/storage-blobs-introduction)
 1. [Create a storage account](/azure/storage/common/storage-account-create)
+1. [Default to and authorize access to blobs using Azure Active Directory](azure/storage/blobs/authorize-access-azure-active-directory)
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-copy-matched-items-get-started.md]:
```suggestion
1. [Default to and authorize access to blobs using Azure Active Directory](/azure/storage/blobs/authorize-access-azure-active-directory)
```

File: microsoft-365/compliance/dlp-copy-matched-items-get-started.md [hunk 3]
```diff
@@ -75,14 +77,16 @@ You can also configure immutability policies for your blob data that protects ag
...
 
 Be sure to save the name and URL of the Azure blob container. To view the URL, open the Azure storage portal \> **Home \> **Storage Accounts** \> **Container** \> **Properties**
+An example of the Azure blob container URL will be in the format `https://storageAccountName.blob.core.windows.net/containerName`.
 
 ### Set permissions on the Azure blob storage
 
-You have to configure two sets of permissions on the blobs, one for the administrators and investigators so they can view and manage evidence and another for users whose devices need to upload items to Azure. You should [create custom role groups in Microsoft Purview compliance](../security/office-365-security/scc-permissions.md) to enforce least privileges and assign accounts to them.
+From the use of Azure Active Directory authorization, you have to configure two sets of permissions on the blobs; one for the administrators and investigators so they can view and manage evidence; and another for users whose devices need to upload items to Azure. You should [create custom role groups in Microsoft Purview compliance](../security/office-365-security/scc-permissions.md) to enforce least privileges and assign accounts to them.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-copy-matched-items-get-started.md]:
```suggestion
Using Azure Active Directory authorization, you have to configure two sets of permissions on the blobs; one for the administrators and investigators so they can view and manage evidence and another for users whose devices need to upload items to Azure. You should [create custom role groups in Microsoft Purview compliance](../security/office-365-security/scc-permissions.md) to enforce least privileges and assign accounts to them.
```

File: microsoft-365/compliance/dlp-copy-matched-items-get-started.md [hunk 4]
```diff
@@ -177,6 +181,7 @@ The JSON for user role group should look like this:
 1. Set how long you want items to be cached on devices if they can't access the Azure storage account. You can choose, **7**, **30**, or **60** days.
 
 1. Select **+ Add storage** and provide the Name and URL of the Azure storage account.
+    1. The URL format is the storage account FQDN/containerName. No SAS token is required as Azure Active Directory authorization is in use.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-copy-matched-items-get-started.md]:
```suggestion
1. Select **+ Add storage** and provide the Name and URL of the Azure storage account. The URL format is the storage account FQDN/containerName. No SAS token is required as Azure Active Directory authorization is in use. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @LucianFrango, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kmkoenen Please merge this PR, Thanks

**Comment by kmkoenen** [comment]
@yogkumgit @JohanFreelancer9 @LucianFrango 

I cannot merge this. These changes were built from the wrong repo. As I explained to Lucian in an email yesterday, in order for me to merge and publish these changes, they must be made in the **micrsoft365-pr** repo, NOT from the **public** repo. [Instructions are in this article](https://learn.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-quick-update?branch=main).

**Comment by kmkoenen** [comment]
@yogkumgit @LucianFrango @JohanFreelancer9 

I cannot merge these changes, they were built from and exist in the wrong repo.  As I explained in an email to Lucian yesterday, they need to be in the **microsoft365-pr** repo, NOT **public*.  To make changes, please follow the [instructions in this [](article](https://review.learn.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-quick-update?branch=main).)

**Comment by LucianFrango** [comment]
Closing this PR and I will resubmit in the correct repo. 
Apologies for the misunderstanding!

**Pull request closed without merging**